src/pages/index.tsx

```diff
@@ -1,15 +1,14 @@
+/**
+ * Note: This file is identical to app.tsx, but app.tsx is the main file.
+ *
+ * The react-router-dom is used to handle all routes after being authenticated.
+ */
+
```

Comment:
why do we need two identical files? can't one just import and use the other if they need to do the same thing?

Reply:
Because of the way Gatsby routes files in pages/* to a url, we do need both files, but they do redirect to the /app url once they load.

I'll update that to be more clear, instead of looking like the same file.